src/main/java/org/veupathdb/service/eda/compute/plugins/correlationassayassay/CorrelationAssayAssayPlugin.java

```diff
@@ -175,6 +175,16 @@ protected void execute() {
             
       
       // Format inputs for R
+      // !!!!!!! TEMPORARY HACK !!!!!!!
+      // This to let the negative values seen in wgcna eigengenes play in mbio-land until we get a better solution
+      // see https://github.com/VEuPathDB/microbiomeComputations/issues/81
+      connection.voidEval("abundanceDataIdColNames <- names(assay1Data)[grepl('stable_id', names(assay1Data))]");
+      connection.voidEval("abundanceDataIds <- assay1Data[,abundanceDataIdColNames, with=FALSE]");
+      connection.voidEval("assay1Data <- plyr::numcolwise(veupathUtils::rescaleToNonNeg)(assay1Data)");
+      connection.voidEval("assay2Data <- plyr::numcolwise(veupathUtils::rescaleToNonNeg)(assay2Data)");
+      connection.voidEval("assay1Data[,abundanceDataIdColNames] <- abundanceDataIds");
+      connection.voidEval("assay2Data[,abundanceDataIdColNames] <- abundanceDataIds");
```

Comment:
did these always have to be the same? It seems like the answer should be yes but wanted to verify.

Reply:
yes